app/views/users/new.html.erb

```diff
@@ -3,7 +3,6 @@
     <%= bootstrap_form_for @user do |f| %>
         <%= f.email_field :email, label: t('common.email') %>
         <%= f.text_field :name, label: t('common.name') %>
-        <%= f.select :role, options_for_select(user_role_options), label: t('user.common.role') %>
```

Comment:
I think some help text here on how to escalate someone's role might be helpful. Maybe something like `New users by default have CM permissions, which let them use the case management workflow. To give someone data access or admin permissions, edit them after you create their accounts here.` I'm spitballing a bit, but I think it'd be helpful.